src/flask/app.py

```diff
@@ -1349,7 +1349,9 @@ def _get_exc_class_and_code(exc_class_or_code):
         else:
             exc_class = exc_class_or_code
 
-        assert issubclass(exc_class, Exception)
+        assert issubclass(
+            exc_class, Exception
+        ), "Custom exceptions should be subclasses of ``Exception``"
```

Comment:
Remove ticks around Exception, this isn't ReST. Add a period at the end of the sentence. Change "should" to "must".